examples/sample-app/README.md

```diff
@@ -462,10 +462,50 @@ Another interesting example is deleting a pod.
 
 Cleaning Up
 -----------
-To clean up all of your environment, you can run the script:
 
-        $ sudo ./cleanup.sh
+### Removing objects
 
-This will stop the `openshift` process, remove files created by OpenShift and kill all Docker containers created by Kubernetes in your host system.  The cleanup script needs root privileges to be able to remove all the directories OpenShift created.
+To empty your test project, deleting all BuildConfigs, ImageStreams,
+DeploymentConfigs, Services, etc, you can use:
 
-**Use with caution!** Any Docker container prefixed with "k8s_" will be killed by this script.
+    $ oc delete all --all
+
+Now you'll have an empty project that you can reuse later.
+Alternatively, you can delete the whole project:
+
+    $ oc delete project test
+
+This will delete the project and all related objects. You'll need to create a
+new project later to deploy more apps.
+
+### Stopping the OpenShift server
+
+If you're running OpenShift as a Docker container, you can stop it now:
+
+    $ docker stop openshift-origin
+
+Otherwise, bring OpenShift's server process to the foreground and kill it:
+
+    $ fg
+    # ^C (CTRL-C)
+
+### Unmounting volumes
+
+*Before removing openshift.local.volumes you'll need to unmount some volumes*
```

Comment:
I don't know how to document this properly... this is the step I was trying to add to the cleanup.sh script back in #2629.